modules/Users/User.php

```diff
@@ -1513,6 +1513,23 @@ public function getEmailLink2(
         $class = ''
     ) {
         $emailLink = '';
+
+        require_once('modules/Emails/EmailUI.php');
```

Comment:
Can you please using `require_once` on top of the file. Also please use `__DIR__` constant as a prefix of file path to make sense where is the file exactly.